apps/worker/src/graph_publisher/graph.publisher.processor.service.ts

```diff
@@ -18,18 +18,35 @@ import { createKeys } from '../../../../libs/common/src/blockchain/create-keys';
 import { ITxMonitorJob } from '../../../../libs/common/src/dtos/graph.notifier.job';
 
 export const SECONDS_PER_BLOCK = 12;
+const CAPACITY_EPOCH_TIMEOUT_NAME = 'capacity_check';
 
+/**
+ * Service responsible for publishing graph updates.
+ */
 @Injectable()
 @Processor(QueueConstants.GRAPH_CHANGE_PUBLISH_QUEUE)
 export class GraphUpdatePublisherService extends BaseConsumer {
+  private capacityExhausted = false;
+
+  public async onApplicationBootstrap() {
+    await this.checkCapacity();
+  }
+
+  // public async onModuleDestroy() {
```

Comment:
This function is part of the code I borrowed from one of the other repos, but our worker structure is slightly different and I couldn't get it to compile.
It would be good to clean up, if we can get it to compile.